npm/janus.d.ts

```diff
@@ -20,7 +20,7 @@ declare namespace JanusJS {
 		Debug = 'debug',
 		Log = 'log',
 		Warning = 'warn',
-		Error = 'error'
+		Error = 'error',
```

Comment:
Why did you add a comma here? It's the last item in the list.

Reply:
`prettier` did it for me( sorry, I'll fix